### PR TITLE
Make "inner invoke" use "remove an event listener"

### LINKED ISSUE #651: Make "inner invoke" use "remove an event listener"
Even though it technically doesn't need to set the removed flag, it seems cleaner to use the same abstraction.

### PR DESCRIPTION
**Pull request by oliverdunk**
My first PR!

A small clarification where inner invoke now uses the "remove an event listener" definition, instead of doing this in its own way.

Resolves #651

***

### :boom: Error: 500 Internal Server Error :boom: ###

[PR Preview](https://github.com/tobie/pr-preview#pr-preview) failed to build. _(Last tried on Jan 15, 2021, 7:33 AM UTC)_.

<details>
<summary>More</summary>

PR Preview relies on a number of web services to run. There seems to be an issue with the following one:

:rotating_light: [CSS Spec Preprocessor](https://api.csswg.org/bikeshed/) - CSS Spec Preprocessor is the web service used to build Bikeshed specs.

:link: [Related URL](https://api.csswg.org/bikeshed/?url=https%3A%2F%2Fraw.githubusercontent.com%2Foliverdunk%2Fdom%2Fd20865635cc2bd2b17ad12c5287b4dd76f615bc1%2Fdom.bs&force=1&md-status=LS-PR&md-Text-Macro=PR-NUMBER%20758)

```
<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>500 Internal Server Error</title>
</head><body>
<h1>Internal Server Error</h1>
<p>The server encountered an internal error or
misconfiguration and was unable to complete
your request.</p>
<p>Please contact the server administrator at 
 [no address given] to inform them of the time this error occurred,
 and the actions you performed just before this error.</p>
<p>More information about this error may be available
in the server error log.</p>
<hr>
<address>Apache/2.4.10 (Debian) Server at api.csswg.org Port 443</address>
</body></html>

```

_If you don't have enough information above to solve the error by yourself (or to understand to which web service the error is related to, if any), please [file an issue](https://github.com/tobie/pr-preview/issues/new?title=Error%20not%20surfaced%20properly&body=See%20whatwg/dom%23758.)._
</details>

### CODE REVIEW

**Comment by annevk** [comment]
Thanks @oliverdunk! I saw your question on IRC about the participation agreement. The WHATWG Steering Group is trying to clarify this in https://github.com/whatwg/sg/issues/67 and I'll ask them there about 1Password for you as it's not super clear to me. I'd assume not, but ...

File: dom.bs [hunk 1]
```diff
@@ -1568,8 +1568,8 @@ run these steps:
    <a for="event listener">capture</a> is true, then <a for=iteration>continue</a>.
 
    <li><p>If <var>listener</var>'s <a for="event listener">once</a> is true, then
-   <a for=list>remove</a> <var>listener</var> from <var>event</var>'s {{Event/currentTarget}}
-   attribute value's <a for=EventTarget>event listener list</a>.
+   <a>remove an event listener</a> with the <var>event</var>'s {{Event/currentTarget}}
+   and <var>listener</var></a>.
    <!-- Do this before invocation to avoid reentrancy issues. No need to set removed to true since
         each listener in listeners is run once anyway. -->
```
Comment (annevk) [inline dom.bs]: I'd reword the second sentence here to say something like "Note that there is no need to set removed to true since listener is run once, but it is done here to share an abstraction."

**Comment by oliverdunk** [comment]
Thanks for the feedback @annevk! I'll try to update this soon, or at the very latest, when we're able to sort the legal situation.

**Comment by annevk** [comment]
This got superseded at some point.
